spras/analysis/ml.py

```diff
@@ -78,6 +78,7 @@ def summarize_networks(file_paths: Iterable[Union[str, PathLike]]) -> pd.DataFra
                 str(tup[0]): 1,
             }, index=tup[1]
         )
+        dataframe = dataframe[~dataframe.index.duplicated(keep='first')]
```

Comment:
add a warning that edges are being dropped from a file

Reply:
If this is hard to do here, we can think about whether it should be handled somewhere else. For instance, should the `parse_output` functions check for duplicate edges and raise a warning or error?